lib/pure/ssl_certs.nim

```diff
@@ -0,0 +1,69 @@
+#
```

Comment:
Hrm, I'd say this module should just be named ``sslcerts``

It's a pretty small module though, maybe it would make sense to stick it into ``net``?

Reply:
CA certificate paths (and more) are not necessarily network-related. People might want to use openssl to perform offline operations (e.g. CAs)

Reply:
Alright. Add this module to Nim's docgen then please.

Reply:
Done.